how-we-work/all-hands-meeting.html.md

```diff
@@ -19,33 +44,17 @@ All Hands events need some work from the meeting organizer both to prepare the m
 to be sure everyone is aware of the meeting and what to expect from it. If you are in charge of
 organizing the All Hands, simply follow these tips to make it a success!
 
-### One day before the meeting
-
+- **Check the calendar event.** We have a shared Nebulab calendar that everyone has joined during
+  their onboarding. In this calendar we have all the meetings events already scheduled. Just be
+  sure information are accurate in the calendar event or change the meeting date if needed.
```

Comment:
The verb usually goes at the end when you are specifying details about the subject, so "Make sure the information in the calendar event is accurate".